lib/exabgp/application/cli.py

```diff
@@ -157,7 +157,7 @@ def write_timeout(signum, frame):
 		sys.exit(0)
 
 	def open_timeout(signum, frame):
-		sys.stderr.write('could not open connection to ExaBGP')
+		sys.stderr.write('could not open connection to ExaBGP\n')
 		sys.stderr.flush()
```

Comment:
I also feel Python shouldn't buffer stderr, but I could be wrong. So I left this for now.